dev-env/lib/dade-common

```diff
@@ -57,7 +57,7 @@ dadeBaseHash() {
 
 # List tools defined in dade
 dadeListTools() {
-    cat $(nix-build $DADE_BASE_ROOT/nix -A dade.tools-list)
+    cat $(nix-build --no-out-link $DADE_BASE_ROOT/nix -A dade.tools-list)
```

Comment:
That one looks fine although I’d also be happy to just kill the command :slightly_smiling_face: